api-interop-layer/mocha.js

```diff
@@ -0,0 +1,19 @@
+import sinon from "sinon";
+import * as mariadb from "mariadb";
+
+export async function mochaGlobalSetup() {
+  sinon.stub(global, "fetch");
+  sinon.stub(mariadb.default, "createConnection");
+}
+
+export async function mochaGlobalTeardown() {
+  global.fetch.restore();
+  mariadb.default.restore();
+}
+
+export const mochaHooks = {
+  beforeEach() {
+    fetch.resetHistory();
+    fetch.resetBehavior();
+  },
+};
```

Comment:
This file provides global stubbing for the `fetch` method as well as the Mariadb `createConnection` method. It also handles resetting them after every test. This way we don't have to handle mocking them everywhere.